docs/manuals/man5/pki_healthcheck.conf.5.md

```diff
@@ -0,0 +1,83 @@
+# pki_healthcheck.conf 5 "June 29, 2020" PKI "PKI Healthcheck Configuration"
+
+## NAME
+
+pki_healthcheck.conf - PKI Healthcheck configuration file.
+
+## DESCRIPTION
+
+The healthcheck.conf configuration file is used to set the defaults when running pki-healthcheck tool.
+
+## SYNTAX
+
+The configuration options are not case sensitive. The values may be case sensitive, depending on the option.
+
+Blank lines are ignored.
+Lines beginning with # are comments and are ignored.
+
+Valid lines consist of an option name, an equals sign and a value. Spaces surrounding equals sign are ignored. An option terminates at the end of a line.
+
+Values should not be quoted, the quotes will not be stripped.
+
+    # Wrong - don't include quotes
+    verbose = "True"
+
+    # Right - Properly formatted options
+    verbose = True
+    verbose=True
+
+Options must appear in the section named *[default]* or *[dogtag]*. There are no other sections defined or used currently.
+
+Options may be defined that are not used. Be careful of misspellings, they will not be rejected.
+
+## OPTIONS
+
+The following options are relevant for the healthcheck tool:
+
+**cert_expiration_days**  
+    The number of days left before a certificate expires to start displaying a warning. The default is 28.
+
+**ca_instance_name**  
+    The name of the PKI instance with CA subsystem. The default is **pki-tomcat**
+
+**kra_instance_name**  
+    The name of the PKI instance with KRA subsystem. The default is **pki-tomcat**
+
+**ocsp_instance_name**  
+    The name of the PKI instance with OCSP subsystem. The default is **pki-tomcat**
+
+**tks_instance_name**  
+    The name of the PKI instance with TKS subsystem. The default is **pki-tomcat**
+
+**tps_instance_name**  
+    The name of the PKI instance with TPS subsystem. The default is **pki-tomcat**
+
+## EXAMPLES
+
+[default]  
+cert_expiration_days = 30  
+
+[dogtag]  
+ca_instance_name = pki-ca  
+kra_instance_name = pki-kra  
+ocsp_instance_name = pki-ocsp  
+tks_instance_name = pki-tks  
+tps_instance_name = pki-tps  
```

Comment:
I think most people generally will only manage a single security domain so these subsystems are likely to be deployed on the same instance. Running multiple Tomcat servers consumes a lot of memory so I don't think people will do that unless it's necessary (e.g. to manage multiple security domains). So maybe we only need a single instance name parameter, for example:

```
[dogtag]
instance_name = pki-tomcat
```

However, in case some people want to deploy multiple instances on the same machine (i.e. for different security domains), maybe we could have a separate section for each instance, probably something like this:
```
[dogtag]
instances = instance1, instance2

[instance1]
instance_name = test1

[instance2]
instance_name = test2
```